config.yaml

```diff
@@ -0,0 +1,28 @@
+# Dependency 'bulma' is brought in as module
+# and mount points are declared
+module:
+  imports:
+  - path: github.com/jgthms/bulma

```

Comment:
A follow-up question regarding this here @deining .

We now pull always the latest `main`/`master` branch from bulma right? 🤔 
Can we also specify a `git tag`? 🤔
Like  `github.com/jgthms/bulma@0.9.3`
I guess this is more stable than always pulling a develop branch?! 🤔

Reply:
> We now pull always the latest `main`/`master` branch from bulma right?

No. we pull in `0.9.4` right now.

The bulma version used is defined in `go.mod`:

https://github.com/StefMa/hugo-fresh/blob/7987c63887d208977c332942cf7a9f756e96cc1d/go.mod#L5

To update bulma inside your theme use this command:

```bash
hugo mod get -u github.com/jgthms/bulma@9734c65c100   # commit hash
```

Instead of a commit, you can also use a version (the tag should have leading v though (e.g. v0.9.4)):

As an example, the command below can be used by users of your theme to update their theme version used:

```bash
hugo mod get -u github.com/StefMa/hugo-fresh@v0.2
```

This only works of course if you start using tags like vn.n. What about tagging this first module version with `v0.1` or so?

Hope that clarifies things!

Reply:
Thank you very much.
Yes, that helped quite a lot.
Its sad that the `go.mod` didn't show the git tag but the hash instead 😭

Reply:
> Its sad that the `go.mod` didn't show the git tag but the hash instead

If the name of the tag in the bulma repo were `v0.9.4` instead of `0.9.4`, you would see it.
So if you want that users of your theme see the tag instead of the hash, but better tag the inital module release with `v1.0`instead of `1.0`.

Reply:
Thank you very much again 🙂 
Here we are: https://github.com/StefMa/hugo-fresh/releases/tag/v1.0.0

With that I have a go.mod and go.sum client like that:
```
// go.mod
require github.com/StefMa/hugo-fresh v1.0.0 // indirect

// go.sum
github.com/StefMa/hugo-fresh v1.0.0 h1:Nxo9WtzfBlaFi1G8NEHQLReJQz+Dn4jRxec+lMrPG6k=
github.com/StefMa/hugo-fresh v1.0.0/go.mod h1:yRzHyco0bfnITjvq7rfAiOADfGMrDlM0kXeKBllUJEc=
github.com/jgthms/bulma v0.0.0-20220508134905-3e00a8e6d0d0/go.mod h1:89FLBKXYFKfOKAoDeUT26V0pHGwzr205cMXAEqtAVOI=
```